be/src/common/config.h

```diff
@@ -268,7 +268,7 @@ CONF_mBool(enable_zonemap_index_memory_page_cache, "false");
 // whether to enable the ordinal index memory cache
 CONF_mBool(enable_ordinal_index_memory_page_cache, "false");
 // whether to disable column pool
-CONF_Bool(disable_column_pool, "true");
+CONF_mBool(disable_column_pool, "true");
```

Comment:
if `disable_column_pool` is updated from `true` to `false`, we should clear the column pool?

Reply:
> if `disable_column_pool` is updated from `true` to `false`, we should clear the column pool?

It does not take effect on the existing column pool, only on the new column pool.